### ISSUE TITLE
loss=nan error

### ISSUE THREAD

**Issue by aj96** [MaybeShewill-CV/lanenet-lane-detection#52]
I ran into a loss=nan error due to labeled images that were all zeros. I had to re-make my train.txt and val.txt to not include any samples that had no lane lines. Is this correct? I would think that the neural net should be able to handle situations where everything is background.

**Comment by MaybeShewill-CV** [comment]
@aj96 The gradients are unstable due to the empty labeled image.

**Comment by MaybeShewill-CV** [comment]
@cardwing Have you ever tried another base net and how well it performed?

**Comment by cardwing** [comment]
@MaybeShewill-CV , I am using an efficient segmentation model as the backbone and testing it in CULane dataset (sorry that I cannot tell the base model). I will release the full model and code after our paper is published. I comment here to just remind other researchers if they meet the same problem which is caused by changing the base net.

**Comment by cardwing** [comment]
@MaybeShewill-CV , besides, thanks for your great work.

**Comment by MaybeShewill-CV** [comment]
@cardwing OK look forward to your new paper work

**Comment by phjhuang** [comment]
@cardwing     Does lanenet  works well  on CUlane dataset???

**Comment by AaronStuart** [comment]
> The solution is that you need to first only train the binary segmentation branch and then jointly train both branches (i.e., binary segmentation & clustering) after the binary segmentation branch is well trained.

@cardwing I want first train the binary segmentation branch too. But i have no idea how to make instance segmentation branch disabled, could you give me some suggestions, thanks :)

**Comment by cardwing** [comment]
Just make the coefficient of the instance segmentation loss be zero.
